src/GHC/DataSize.hs

```diff
@@ -33,7 +37,7 @@ wORD_SIZE = SIZEOF_HSWORD
 
 -- | Calculate size of GHC objects in Bytes. Note that an object may not be
 --   evaluated yet and only the size of the initial closure is returned.
-closureSize :: Num b => a -> IO b
+closureSize :: a -> IO Word
```

Comment:
Do I understand it correctly that previously you could call it with `let bar <- closureSize foo :: IO Word` previously but now it's forced to be Word? I guess Word is the only type that makes sense since that's the limit of how much memory will be used.